lib/nvidia_utils.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: NVIDIA helper functions
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+package nvidia_utils;
+
+use Exporter;
+use testapi;
+use strict;
+use warnings;
+use utils;
+
+our @EXPORT = qw(
+  install
+  validate
+);
+
+sub _add_repos
+{
+    zypper_ar(get_required_var('NVIDIA_REPO'), name => 'nvidia', no_gpg_check => 1, priority => 90);
+    zypper_ar(get_required_var('NVIDIA_CUDA_REPO'), name => 'cuda', no_gpg_check => 1, priority => 90);
+}
+
+=head2 install
+
+ install([ variant => 'cuda' ]);
+
+Install the NVIDIA driver and the compute utils, making sure to remove
+any conflicting variant first. Also, it tries to add the relevant
+repositories to grab the packages from, defined by the job through
+NVIDIA_REPO and NVIDIA_CUDA_REPO.
+
+=cut
+sub install
+{
+    my $variant_std = 'nvidia-open-driver-G06-signed-kmp-default';
+    my $variant_cuda = 'nvidia-open-driver-G06-signed-cuda-kmp-default';
+    my %args = @_;
+    my $variant = $args{variant} eq "cuda" ? $variant_cuda : $variant_std;
+
+    _add_repos();
```

Comment:
I think we don't need separate function for it, just place  those two `zypper_ar` here.

Reply:
Indeed. Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22153/commits/c177d7614389a6c20f83a2affd1aeb4b4bfa96b0. BTW, new VR: https://rmarliere-openqa.qe.prg2.suse.org/tests/1240